core/templates/pages/blog-dashboard-page/blog-post-editor/blog-post-editor.component.html

```diff
@@ -88,6 +89,14 @@ <h6 class="mt-1">{{'I18N_BLOG_POST_EDITOR_SHOW_PREVIEW_HEADING' | translate | up
           <em [innerHTML]="'I18N_BLOG_POST_EDITOR_TITLE_MIN_LENGTH_ERROR' | translate: { minChars: MIN_CHARS_IN_BLOG_POST_TITLE }">
           </em>
         </span>
+        <span class="oppia-input-box-subtitle" *ngIf="!isTitlePatternValid()">
+          <em [innerHTML]="'I18N_BLOG_POST_EDITOR_TITLE_INVALID_CHARACTERS_ERROR' | translate">
+          </em>
```

Comment:
```suggestion
          <em>
              {{ 'I18N_BLOG_POST_EDITOR_TITLE_INVALID_CHARACTERS_ERROR' | translate }}
          </em>
```

Can we something like this? (here and elsewhere)

Reply:
Done